src/coreComponents/meshUtilities/PerforationData.cpp

```diff
@@ -78,8 +76,8 @@ void DecideWellDirection( VEC_TYPE const & vecWellElemCenterToPerf,
                           real64 & k2 )
 {
   // vertical well (approximately along the z-direction)
-  if( fabs( vecWellElemCenterToPerf[2] ) > fabs( vecWellElemCenterToPerf[0] )
-      && fabs( vecWellElemCenterToPerf[2] ) > fabs( vecWellElemCenterToPerf[1] ) )
+  if( fabs( vecTopToBottom[2] ) > fabs( vecTopToBottom[0] )
```

Comment:
I would rename `vecTopToBottom` to `topToBottom`. Or at least `topToBottomVec`?

Reply:
Done, I renamed it to `topToBottomVec`